<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionFIeld</class>
 <widget class="QWidget" name="QuestionFIeld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="answer_button">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
      <property name="text">
       <string>Answer</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="like_button">
        <property name="styleSheet">
         <string notr="true">color:black</string>
        </property>
        <property name="text">
         <string>:)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="dislike_button">
        <property name="styleSheet">
         <string notr="true">color:black</string>
        </property>
        <property name="text">
         <string>:(</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>207</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="question">
      <property name="styleSheet">
       <string notr="true">color:black</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: black</string>
        </property>
        <property name="text">
         <string>Answer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="answer_line">
        <property name="styleSheet">
         <string notr="true">color:black</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
